docs/docs/APIdocs/spark.ml/DLClassifier.md

```diff
@@ -0,0 +1,35 @@
+## DLClassifier ##
+
+**Scala:**
+```scala
+val classifier = new DLClassifier[Float](
+    val model: Module[Float],
+    val criterion : Criterion[Float],
+    val featureSize : Array[Int]
```

Comment:
Some general rules in instance creation example here. 
- Specify default values if any.  
- Remove paramters type. If user copy/paste this line into his code, it shouldn't cause compile error.  For exact parameter types user should be able to find it in scala API docs
- Explain paramters in details in description if you think they need elaboration.